src/fidesops/util/oauth_util.py

```diff
@@ -139,7 +140,7 @@ async def verify_oauth_client(
         raise AuthorizationError(detail="Not Authorized for this action")
 
     client = ClientDetail.get(
-        db, object_id=client_id, config=config, scopes=security_scopes.scopes
+        db, object_id=client_id, config=config, scopes=SCOPE_REGISTRY
```

Comment:
Is there already a ticket to cut over to using the fideslib version?

Reply:
We're bumping version in this PR here- https://github.com/ethyca/fidesops/pull/830/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552L13

Reply:
You mean fideslib scops? We weren't able to put the scopes in fideslib and had to do it this way because the different libraries have different scopes.

Reply:
I meant using the fideslib oauth utils